p2p/src/error.rs

```diff
@@ -40,7 +39,6 @@ define_error! {
             | _ | { "public key missing" },
 
         Signature
-            [ DisplayOnly<SignatureError> ]
```

Comment:
This error is deliberately opaque, but we can't make the `Display` impl transparent in flex-error, so we have our own message. OK.